src/Relation/Binary/Morphism/Construct/Product.agda

```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The projection morphisms for relational structures arising from the
+-- product construction
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+module Relation.Binary.Morphism.Construct.Product where
+
+import Data.Product.Base as Product using (<_,_>; proj₁; proj₂)
+open import Data.Product.Relation.Binary.Pointwise.NonDependent as Pointwise
+  using (Pointwise)
+open import Level using (Level)
+--open import Relation.Binary.Construct using ()
+open import Relation.Binary.Bundles.Raw using (RawSetoid)
+open import Relation.Binary.Morphism.Structures using (IsRelHomomorphism)
+
+private
+  variable
+    a b c ℓ₁ ℓ₂ ℓ : Level
+
+
+------------------------------------------------------------------------
+-- product construction on RawSetoid belongs in Relation.Binary.Construct.Product?
```

Comment:
Is this comment worth following up on, so that the construction of `P` is moved to such a destination, and then re-imported here?

Reply:
As above: possible downstream refactoring!